core/liquidity/snapshot.go

```diff
@@ -320,6 +347,29 @@ func (e *SnapshotEngine) serialiseSupplied() ([]byte, bool, error) {
 	return e.marshalPayload(payload)
 }
 
+func (e *SnapshotEngine) serialiseScores() ([]byte, bool, error) {
+	scores := make([]*snapshotpb.LiquidityScore, 0, len(e.avgScores))
+	for k, v := range e.avgScores {
+		s := &snapshotpb.LiquidityScore{
+			PartyId: k,
+			Score:   v.String(),
+		}
+		scores = append(scores, s)
+	}
+
+	payload := &snapshotpb.Payload{
```

Comment:
Can you sort the `scores` slice by `PartyID` before putting it in th payload? It just makes sure the snapshot is deterministic.

Reply:
you made it sound like it was optional lol, I was more brutal :)